calico/maintenance/monitor/monitor-component-metrics.md

```diff
@@ -229,6 +353,65 @@ We can configure Prometheus using a ConfigMap to persistently store the desired
 > **Note**: A comprehensive guide about configuration file can be [found at this link](https://prometheus.io/docs/prometheus/latest/configuration/configuration/){:target="_blank"}.
    {: .alert .alert-info}
 
+{% tabs %}
+<label:Operator,active:true>
+<%
+```bash
+kubectl apply -f - <<EOF
```

Comment:
What's different about this config for operator vs. manifest?

Reply:
I've added a drop `calico-typha` metrics. 
https://github.com/projectcalico/calico/pull/5417/files/557d5fdd8115b62e93930a05fce6c4aaf31ef539#diff-85ac17257057f1fba23342ddaf0763ef5ec9b6927b6467e89c8a3ef3c4102f3eR397-R399
Changed the kubernetes service name
https://github.com/projectcalico/calico/pull/5417/files/557d5fdd8115b62e93930a05fce6c4aaf31ef539#diff-85ac17257057f1fba23342ddaf0763ef5ec9b6927b6467e89c8a3ef3c4102f3eR407